android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -411,9 +411,7 @@ public boolean onMenuOrKeyboardAction(int id, boolean fromMenu) {
             openPlaylist(true);
         } else if (id == R.id.add_to_playlist_id) {
             BraveToolbarLayoutImpl layout = getBraveToolbarLayout();
-            if (layout != null) {
```

Comment:
was that check removed because it could never happen? I think there were cases when toolbar could be null. I could be wrong though.

Reply:
Android Studio Lint highlighted that was always non null. If it was a false positive and there are cases where it could be null I'm happy reintroduce the check, better be extra safe.

Reply:
I don't remember honestly did we have any cases or not, probably we can remove as there is an assert as well. I'll keep an eye on crashes after that PR is merged.